modules/caddyhttp/vars.go

```diff
@@ -25,6 +25,7 @@ import (
 func init() {
 	caddy.RegisterModule(VarsMiddleware{})
 	caddy.RegisterModule(VarsMatcher{})
+	caddy.RegisterModule(MatchVarRE{})
```

Comment:
nit: It seems like it would be more consistent to name this `RegexVarsMatcher` and a little clearly to spell out `Regex` IMO

Reply:
Overlooked the collapsed conversation below, looks like you've already thought specifically about the naming. 👍